firebase.ts

```diff
@@ -0,0 +1,20 @@
+// Import the functions you need from the SDKs you need
+import { initializeApp } from "firebase/app";
+// import { getAnalytics } from "firebase/analytics";
+import { getAuth } from "firebase/auth";
+import { firebaseConfig } from "./.secrets/firebase-config";
```

Comment:
is it possible to use the existing pmc-portal-credential.json file to authenticate and initialize firebase?